.husky/commit-msg

```diff
@@ -0,0 +1 @@
+pnpm commitlint --edit $1
```

Comment:
Husky has to be updated in the process. It has now files here instead of a global config file.